tracee-ebpf/tracee/tracee.go

```diff
@@ -1037,10 +1075,23 @@ func (t *Tracee) updateFileSHA() {
 	}
 }
 
-func (t *Tracee) invokeInitNamespacesEvent() {
+// invokeManualTriggeredEvents handles a self triggered events which aren't created/started from the kernel
+func (t *Tracee) invokeManualTriggeredEvents() {
+
+	//invoke InitNamespacesEvent
 	if t.eventsToTrace[InitNamespacesEventID] {
 		systemInfoEvent, _ := CreateInitNamespacesEvent()
 		t.config.ChanEvents <- systemInfoEvent
 		t.stats.eventCounter.Increment()
 	}
+	// invoke DetectHookedSyscallsEvent
+	if t.eventsToTrace[DetectHookedSyscallsEventID] {
+		path := "/tmp/tracee/out/tracee.pid"
```

Comment:
output path doesn't have to be /tmp/tracee/out.
Please use `cfg.Capture.OutputPath` instead